File: CursoJava/src/Exer/Exer02.java
package Exer;

import java.util.Scanner;

public class Exer02 {

	public static void main(String[] args) {
	
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Qual o raio do círculo: ");
		double raio = sc.nextDouble();
		double pi = 3.14159;		
		double area = pi * (raio * raio);

		System.out.printf("O valor da área é: %.4f%n", area);
		
		sc.close();
	}

}

File: CursoCSharp-Udemy/exer/udemy/EstruturaSequencial/Exer02.cs
using System;

namespace exer.udemy.EstruturaSequencial
{
    public class Exer02
    {
        static void Main(string[] args)
        {
            double pi = 3.14159;

            Console.Write("Informe o valor do raio do círculo: ");
            double radius = double.Parse(Console.ReadLine());

            double area = pi * (radius * radius);

            Console.WriteLine($"Resultado: {area}");
        }
    }
}